crates/ra_assists/src/add_explicit_type.rs

```diff
@@ -69,6 +69,21 @@ mod tests {
         );
     }
 
+    // https://github.com/rust-analyzer/rust-analyzer/issues/1592
+    #[test]
+    fn add_explicit_type_infers_correct_type_for_floating_point_literal() {
```

Comment:
I think this would be more fitting to test in the type inference tests, i.e. `ra_hir/hy/tests.rs`.